ui/pages/notifications/notification-components/erc1155-sent-received/erc1155-sent-received.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { NotificationServicesController } from '@metamask/notification-services-controller';
 import { CHAIN_IDS } from '../../../../../shared/constants/network';
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
 import { t } from '../../../../../app/scripts/translate';
```

Comment:
What is the recommend pattern to retrieve this translation function for areas that are not components or hooks?

(The below component objects can be converted to use hooks, but just something to raise)

Reply:
There isn't a pattern for that yet. Traditionally UI translations are done via:

```
const t = useI18nContext();
```

If we definitely want static translations like this, then one option is to move the `t` function itself to `shared` (most of the infrastructure behind it is already shared).

Reply:
OK agreed. Context on our side is that push notifications also undergo translations.
This is a little tricky since I don't want to bloat the background by dragging in all the translation files.

(Future) maybe we can have smaller translation subfiles that can be brought in the background rather than the all translations.

Reply:
To clarify, translation was originally only possible in the UI via the React context.

The `t` function we're currently using here was added specifically to allow the background code to translate, and in fact the original use case was also so we could translate the browser notifications for transactions.

Reply:
Yep awesome awesome!

Reply:
Resolving this convo 😄